verticapy/toolbox.py

```diff
@@ -723,6 +723,19 @@ def nearest_column(columns: list, column: str):
                 result = (elem, d)
     return result
 
+# ---#
+def ooe_details_transform(L: list):
```

Comment:
Can you please add comments explaining the function.

Reply:
Done;